build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/AndroidCompose.kt

```diff
@@ -36,7 +36,7 @@ internal fun Project.configureAndroidCompose(
         }
 
         composeOptions {
-            kotlinCompilerExtensionVersion = libs.findVersion("androidxCompose").get().toString()
+            kotlinCompilerExtensionVersion = libs.findVersion("androidxComposeCompiler").get().toString()
```

Comment:
Why this instead of

```
    composeOptions {
        kotlinCompilerExtensionVersion = libs.versions.androidxComposeCompiler.get()
    }
```

Reply:
`libs` here is a [`VersionCatalog`](https://docs.gradle.org/7.4.2/javadoc/org/gradle/api/artifacts/VersionCatalog.html), there's no `versions` available on it